### DISCUSSION TITLE
OracleDB dashboard

### DISCUSSION THREAD

**Discussion by Ferrox12** [dbeaver/dbeaver#9151] [Q&A] [answered]
Hi,
where can I find some OracleDB dashboard for DBeaver?
In "Manage Dashboard" there are only Exasol, MySQL and PostgreSQL.
Is there a DBeaver plugin with specific queries for db administrator with related dashboard?

**Comment by uslss** [comment, marked as answer]
Hi @Ferrox12 ,
Oracle default dashboards are available in EE version only.
BTW, you could create them yourself (https://github.com/dbeaver/dbeaver/wiki/Dashboards#creating-dashboards)
BR
